vyper/parser/memory_allocator.py

```diff
@@ -0,0 +1,23 @@
+from typing import (
+    Tuple,
+)
+
+from vyper.utils import (
+    MemoryPositions,
+)
+
+
+class MemoryAllocator():
+
```

Comment:
We should probably also add an instance property type annotation here:
```suggestion
next_mem: int

```